src/resources/gpio_esp32.cc

```diff
@@ -38,6 +38,8 @@
 
 namespace toit {
 
+static const char* const RESTRICTED_PIN_ERROR = "RESTRICTED";
```

Comment:
Can we use one of the predefined / preallocated ones instead?

Reply:
Switched to "PERMISSION_DENIED".
done.